ui/redux/selectors.js

```diff
@@ -392,6 +392,10 @@ export const getLocusListTableData = createSelector(
       data = data.filter(locusList => !locusListGuids.includes(locusList.locusListGuid))
     }
 
+    data = data.map(({ items, ...locusList }) => (items ?
+      { geneNames: items.map(({ gene }) => (gene || {}).geneSymbol), ...locusList } :
+      locusList))
```

Comment:
no because the way it currently is it falls back to the existing `geneNames` when there are no `items`, and this would overwrite the `geneNames` with an empty list